src/machine/machine_rp2040_pwm.go

```diff
@@ -278,8 +278,14 @@ func (pwm *pwmGroup) setPeriod(period uint64) error {
 	rhs := 16 * period / ((1 + phc) * periodPerCycle * (1 + topStart)) // right-hand-side of equation, scaled so frac is not divided
 	whole := rhs / 16
 	frac := rhs % 16
-	if whole > 0xff {
+	switch whole {
+	case 0xff:
```

Comment:
This isn't equivalent?  This won't clamp 0x100 since it's not exactly 0xff?

Reply:
darn. I was about to break it even more. Sorry, I'm juggling university work and tinygo in a dangerous mix.